treeherder/model/error_summary.py

```diff
@@ -42,18 +57,32 @@ def get_error_summary(job, queryset=None):
     # cache terms generated from error line to save excessive querying
     term_cache = {}
 
-    error_summary = [bug_suggestions_line(err, term_cache) for err in queryset]
+    error_summary = []
+    # Future suggestion, set this to queryset[:10] to reduce calls to bug_suggestions_line
+    for err in queryset:
+        summary, line_cache = bug_suggestions_line(
+            err, logdate=job.submit_time, term_cache=term_cache, line_cache=line_cache
+        )
+        error_summary.append(summary)
 
     try:
         cache.set(cache_key, error_summary, BUG_SUGGESTION_CACHE_TIMEOUT)
     except Exception as e:
         newrelic.agent.record_custom_event('error caching error_summary for job', job.id)
         logger.error('error caching error_summary for job %s: %s', job.id, e, exc_info=True)
 
+    try:
+        cache.set(
+            line_cache_key, line_cache, BUG_SUGGESTION_CACHE_TIMEOUT * LINE_CACHE_TIMEOUT_DAYS
```

Comment:
This should not rely on `BUG_SUGGESTION_CACHE_TIMEOUT` being a day but use a separate constant for this.